### DISCUSSION TITLE
Unable to enable Vault in database

### DISCUSSION THREAD

**Discussion by thestepafter** [supabase/supabase#9022] [Questions] [answered]
I am following this documentation [https://supabase.com/blog/supabase-vault](https://supabase.com/blog/supabase-vault).

but the command:

`create extension vault with schema vault;`

is throwing the following error:

**[58P01] ERROR: could not open extension control file "/var/lib/postgresql/extension/vault.control": No such file or directory**

I haven't been able to find any other documentation on enabling Vault and I don't see it on the database extensions page in the UI.

**Comment by Hallidayo** [comment, marked as answer]
Hi @thestepafter,

Looking at the blog post I don't think Supabase Vault is available on the platform yet:

`Vault is not available on the Supabase platform yet. We're finalizing usability and security testing.`

**Comment by thestepafter** [comment]
Thank you @Hallidayo, I was hoping that since it was a blog post from about a month ago it would be available.

**Reply by Hallidayo** [reply]
@thestepafter - you coud reach out to Supabase via email: [support@supabase.io](mailto:support@supabase.io) for more information.

**Comment by tobyab** [comment]
Asking for a quick update! Is Vault available yet? I've been trying to find it within my project settings, but to no avail.

**Comment by Yihwan** [comment]
I think Vault is in beta, but still running into this error locally: 

```
Applying migration 20230603234645_remote_commit.sql...
Error: ERROR: could not open extension control file "/usr/share/postgresql/14/extension/supabase_vault.control": No such file or directory (SQLSTATE 58P01)
At statement 19: --                                                                 
-- Name: supabase_vault; Type: EXTENSION; Schema: -; Owner: -      
--                                                                 
                                                                   
CREATE EXTENSION IF NOT EXISTS "supabase_vault" WITH SCHEMA "vault"
```

**Reply by zappellinn** [reply]
I solved it by changing my postgres version in the `config.toml` in the `[db]` section to version 14

**Comment by blackmad** [comment]
I'm confused how I'm supposed to have a local env that looks like my remote env with this vault issue?

... okay commenting out that extension seemed to let it just work

**Reply by corepay** [reply]
What does this mean?  Comment out that/what extension makes it work?
